packages/shared/lib/services/sync/job.service.ts

```diff
@@ -76,7 +76,7 @@ export const getLatestSyncJob = async (sync_id: string): Promise<SyncJob | null>
 };
 
 export const getSyncJobByRunId = async (run_id: string): Promise<SyncJob | null> => {
-    const result = await schema().from<SyncJob>(SYNC_JOB_TABLE).where({ run_id, deleted: false }).first();
+    const result = await db.knex.from<SyncJob>(SYNC_JOB_TABLE).select('*').where({ run_id }).first();
```

Comment:
have we already removed all the `deleted` sync_jobs. if not this query is potentiall gonna return a different row

Reply:
we are not using this feature so it's not possible that it returns a different thing

Reply:
you mean it is never set to true? because `getSyncJobByRunId` is called

Reply:
yes `deleted` is never set to true